ravenpy/config/rvs.py

```diff
@@ -577,7 +595,12 @@ def add_nc_variable(self, **kwargs):
             cmd = GriddedForcingCommand(**kwargs)
 
         if isinstance(self._var_cmds.get(std_name, None), dict):
-            self._var_cmds[std_name] = replace(cmd, **self._var_cmds[std_name])
+            d = self._var_cmds[std_name]
+            lt = d.pop("linear_transform", None)
+            if lt:
+                d["scale"] = lt[0]
+                d["offset"] = lt[1]
+            self._var_cmds[std_name] = replace(cmd, **d)
```

Comment:
I thought I had converted all instances of linear_transform to scale and offset.

Reply:
Not in the tests it seems: https://github.com/Ouranosinc/raven/search?q=linear_transform

Reply:
Sorry I'm confused: I mean in this repo yes, but in RavenWPS there are some remaining.

Reply:
I removed it